src/OpenApi/src/Schemas/OpenApiSchemaKeywords.cs

```diff
@@ -12,7 +12,6 @@ internal class OpenApiSchemaKeywords
     public const string AnyOfKeyword = "anyOf";
     public const string EnumKeyword = "enum";
     public const string DefaultKeyword = "default";
-    public const string NullableKeyword = "nullable";
```

Comment:
How can we get rid of "nullable" and still support OpenAPI 3.0 ? Maybe this "nullable" is not the "nullable" that we add to schemas for 3.0 ?

Reply:
OpenAPI.NET will write out the nullable: true property if it encounters a `null` value in the `type` property based on the logic here: https://github.com/microsoft/OpenAPI.NET/blob/e917e4cb89243e9345899a1118f42dd5b8845ee2/src/Microsoft.OpenApi/Models/OpenApiSchema.cs#L629-L630

That means that end-users will not be able to inspect the Nullable property of the schema to see if it is null. Instead, they have to go see if there is `JsonSchemaType.Null` in the `OpenApiSchema.Type` property.